recipes/qtsass/meta.yaml

```
@@ -0,0 +1,44 @@
+{%- set name = "qtsass" -%}
+{%- set version = "0.1.1" -%}
+{%- set hash = "4b1491e9810c7216365647e416fd7b49aeadc1ac25add8c6131deddf7549a072" -%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [win and py2k]
+
+requirements:
+  host:
+    - python
```

If this becomes `noarch` you can constrain the Python version with:
```suggestion
    - python >=3
```